microcosm_pubsub/chain/chain.py

```diff
@@ -63,6 +63,8 @@ def __call__(self, context=None, **kwargs):
         context = context or self.new_context_type()
         context.update(kwargs)
 
+        res = None
```

Comment:
Technically, if there are no `self.links`, this var could be undefined when we try and return out. Default to null instead.